common/taskana-common-data/src/main/resources/sql/test-data/workbasket.sql

```diff
@@ -1,11 +1,11 @@
 -- KSC workbaskets
```

Comment:
Please add/remove spaces so that after your changes all lines are aligned to this header.